backend/src/services/wallet_service.py

```
@@ -74,28 +96,45 @@ async def get_wallet_info(self) -> Optional[Dict]:
-            # TODO: Fetch USD value for SOL and tokens
-            usd_value = sol_balance * 0 # Placeholder for actual SOL price
+            # Fetch real SOL price from Jupiter Price API
+            sol_price = 0.0
+            try:
+                import httpx
+                async with httpx.AsyncClient() as client:
+                    price_response = await client.get("https://api.jup.ag/price/v2?ids=So11111111111111111111111111111111111111112")
+                    if price_response.status_code == 200:
+                        price_data = price_response.json()
+                        sol_price = float(price_data['data']['So11111111111111111111111111111111111111112']['price'])
+            except Exception as pe:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)